tutorials/scripting/scene_tree.rst

```diff
@@ -141,14 +141,14 @@ Changing current scene
 
 After a scene is loaded, you may want to change this scene for
 another one. One way to do this is to use the
-:ref:`SceneTree.change_scene() <class_SceneTree_method_change_scene>`
+:ref:`SceneTree.change_scene_to_file() <class_SceneTree_method_change_scene>`
```

Comment:
You also need to update the `<class_SceneTree_method_…>` part or else the link won’t work.